scipy/special/_add_newdocs.py

```diff
@@ -792,6 +792,21 @@ def add_newdoc(name, doc):
     function by multiplying the result of ``betainc(a, b, x)`` by
     ``beta(a, b)``.
 
+    ``betainc(a, b, x)`` is treated as a two parameter family of functions
+    of a single variable `x`, rather than as a function of three variables.
+    This impacts only the limiting cases ``a = 0``, ``b = 0``, ``a = inf``,
+    ``b = inf``.
```

Comment:
Want to add `inf` cases to the tests?

Reply:
Yes, that's an oversight. Good suggestion.